src/lib/circuit_value.ts

```diff
@@ -434,6 +434,7 @@ function circuitMain(
 let primitives = new Set(['Field', 'Bool', 'Scalar', 'Group']);
 let complexTypes = new Set(['object', 'function']);
 
+// TODO: Should `ProvableExtension` use `any` as well?
```

Comment:
@mitschabaude Should we update `ProvableExtension` and all related types to use `any` instead of `JSONValue` as well?

Reply:
I agree! Let's merge it